### ISSUE TITLE
`pip install coala-bears` installs PyYAML==3.12, and then complains that it needs PyYAML==3.11 when running

### ISSUE THREAD

**Issue by underyx** [coala-analyzer/coala-bears#731]
@coala-analyzer/coala-admins probably urgent issue

Active bears in the config are:
```
bears =
    SpaceConsistencyBear,
    FilenameBear,
    LineLengthBear,
    LineCountBear,
    KeywordBear,
    InvalidLinkBear,
    PyImportSortBear,
    PyUnusedCodeBear,
    RadonBear,
    PEP8Bear,
    GitCommitBear,
```

**Comment by chauffer** [comment]
:+1:

This is the error I'm getting:
`[ERROR][15:33:43] There is a conflict in the version of a dependency you have installed and the requirements of coala. This may be resolved by creating a separate virtual environment for coala or running 'pip install PyYAML==3.11'. Be aware that the latter solution might break other python packages that depend on the currently installed version`

**Comment by sils1297** [comment]
@hypothesist this needs a backport, didn't we check this?

**Comment by sils1297** [comment]
see also https://github.com/Kuniwak/vint/issues/162#issuecomment-243236874

**Comment by underyx** [comment]
Fixed in vim-vint 0.3.10 with https://github.com/Kuniwak/vint/pull/177
